tests/superphot_plus/test_lightcurve.py

```diff
@@ -177,20 +178,71 @@ def test_sort_copy():
     assert np.all(lc.bands == bands)
 
 
+def test_filter_bands():
+    times = np.array(range(9))
```

Comment:
```suggestion
    times = np.arange(9)
```

Here and elsewhere below. It's not much of an improvement, but just a nifty little shortcut.

Reply:
That is better. Thanks.